src/stores/video.ts

```diff
@@ -35,7 +46,9 @@ export const useVideoStore = defineStore('video', () => {
   const { availableStreams } = mainWebRTCManager.startStream(ref(undefined), allowedIceIps)
   const availableIceIps = ref<string[]>([])
   const videoRecoveryWarningAlreadyShown = useStorage('video-recovery-warning-already-shown', false)
-  const unprocessedVideos = useStorage<{ [key in string]: UnprocessedVideoInfo }>('cockpit-unprocessed-video-info', {})
+  const unprocessedVideos = useStorage<{
+    [key in string]: UnprocessedVideoInfo
+  }>('cockpit-unprocessed-video-info', {})
```

Comment:
It wasn't changed manually. This was ESLint auto formatting on save. 
A potentially unneeded change, since the original line doesn't take more than 80 chars.

Reply:
Yeah, strange.

Here it is not linting the original line.

I would guess `eslint` is using the default configuration on your machine, instead of the ones on the repository (which allow longer lines).

Reply:
Nice, I'll check that

Reply:
This one is still missing.